From 07b5998037acbaa274fa72c17965de7a2889df60 Mon Sep 17 00:00:00 2001
From: Amazon Q <q@amazon.com>
Date: Mon, 4 Aug 2025 16:00:00 +0000
Subject: [PATCH] Fix missing semicolon in crc32.c

Add missing semicolon after return statement in crc_step function.

Upstream-Status: Submitted [https://github.com/aws-greengrass/aws-greengrass-sdk-lite/pull/25]

Signed-off-by: Amazon Q <q@amazon.com>
---
 src/crc32.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/crc32.c b/src/crc32.c
index 0414d76..152779d 100644
--- a/src/crc32.c
+++ b/src/crc32.c
@@ -20,7 +20,7 @@
 #if HAS_BUILTIN_CRC
 
 static uint32_t crc_step(uint32_t crc, uint8_t byte) {
-    return __builtin_rev_crc32_data8(crc, byte, 0x04C11DB7L)
+    return __builtin_rev_crc32_data8(crc, byte, 0x04C11DB7L);
 }
 
 #else
